pkg/metrics/grafana/tidb.json

```
@@ -14709,7 +14709,104 @@
             "align": false,
             "alignLevel": null
           }
-	}
+	},
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "Cached table count of infoschema cache v2",
+          "fieldConfig": {
+            "defaults": {},
+            "overrides": []
+          },
+          "fill": 1,
```

```suggestion
          "fill": 0,
```

unnecessary to fill the color.